index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Tribute Page</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <main id="main">
+    <h1 id="title">Dr. Norman Borlaug</h1>
+
```

Comment:
Kindly remind you to be careful with blank lines, spaces, and indentations. Make sure that you apply semantic HTML & CSS best [practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)